consensus/spos/bls/subroundStartRound.go

```diff
@@ -196,8 +196,8 @@ func (sr *subroundStartRound) initCurrentRound() bool {
 		}
 		sr.AppStatusHandler().SetStringValue(common.MetricConsensusState, "not in consensus group")
 	} else {
-		isLeader := leader == sr.SelfPubKey() && sr.ShouldConsiderSelfKeyInConsensus()
-		if !isLeader && !sr.IsKeyManagedByCurrentNode([]byte(leader)) {
+		isLeader := leader == sr.SelfPubKey()
+		if !isLeader && sr.ShouldConsiderSelfKeyInConsensus() {
```

Comment:
I guess we should test that the current node had at least one managed key in the consensus group.

Reply:
updated